ui/.eslintrc.js

```diff
@@ -166,7 +166,6 @@ module.exports = {
     indent: "off",
     "jsdoc/check-alignment": "error",
     "jsdoc/check-indentation": "error",
-    "jsdoc/newline-after-description": "error",
```

Comment:
Is this something we control or auto-generated?

Reply:
that check's gone somehow :shrug: